app/src/test/java/org/greenstand/android/TreeTracker/models/location/LocationDataCapturerTest.kt

```diff
@@ -200,4 +200,4 @@ class LocationDataCapturerTest {
             locationDataCapturer.isConvergenceWithinRange()
         )
     }
-}
```

Comment:
Can you change your IDE settings to avoid adding this to each file you touch?

Reply:
Should be good now